docs/application/web/api/5.5/device_api/mobile/tizen/content.html

```diff
@@ -675,6 +675,10 @@ <h4>Methods</h4>
 <p><span class="privilege">Privilege: </span>
  http://tizen.org/privilege/content.read
             </p>
+<p><span class="remark">Remark: </span>
+ Attributes available for <em>Content</em> objects filtering are listed in
+<a href="https://developer.tizen.org/development/guides/web-application/data-storage-and-management/data-filtering-and-sorting#content_filter">Data Filtering and Sorting guide</a>.
```

Comment:
Question to the "docs/application/web/guides/data/data-filter.md#content-filter-attributes" file.

In this file we have "Content filter attributes" table. Don't we need to add "isFavorite" to that table?
Also, in this "Table: Content filter attributes", we have "modifedDate" attribute.
I'm wondering whether if "modifiedDate" isn't correct spelling.
Would you please check?

If I understood something wrong, please let me know.

Reply:
Hello, @rwalczyna, would you please check my comment? If you think current state is OK to merge, would you please confirm it?

Reply:
@mijin85cho sorry for late response, we were checking your question.
1. I fixed the typo you mentioned.
2. For now, filtering by isFavorite members is not available, thus it cannot be added in the table related to content filtering.

Reply:
Thanks for the answer and the update.
@rwalczyna If we have no more issues with this PR, would you please re-approve it?

Reply:
For me it also looks okay.